library/deletedrug.php

```diff
@@ -27,11 +27,46 @@
     if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
         CsrfUtils::csrfNotVerified();
     }
+
+    /**
+     * find the drug name in the prescription table
+     */
+    try {
+        $drug_name = "SELECT patient_id, drug FROM prescriptions WHERE id = ?";
+        $dn = sqlQuery($drug_name, [$id]);
+    } catch (Exception $e) {
+        echo 'Caught exception ', text($e->getMessage()), "\n";
+        if ($e->getMessage()) {
+            exit;
+        }
+    }
+
+    /**
+     * remove drug from the medication list if exist
+     */
+    try {
+        $pid = $dn['patient_id'];
+        $drugname = $dn['drug'];
+        $medicationlist = "DELETE FROM lists WHERE pid = ? AND type = ? AND title = ?";
+        sqlQuery($medicationlist, [$pid, 'medication', $drugname]);
```

Comment:
no reason to bind a constant and better to use sqlStatement here. this should be:
```
$medicationlist = "DELETE FROM lists WHERE pid = ? AND type = 'medication' AND title = ?";
sqlStatement($medicationlist, [$pid, $drugname]);
```
Also, do a check to ensure $dn['drug'] is not empty in order to avoid wiping out medications without a title (this shouldn't happen in theory, but maybe there is an instance out there that is using codes and not titles).